### PR TITLE
fix: remove AWS/ElasticInterface service since it doesn't seem supported (anymore?)

### PR DESCRIPTION
**Pull request by Tulux**
AWS/ElasticInterface is now refused by Splunk API which blocks the integration
+ it is not listed in https://docs.splunk.com/observability/en/gdi/integrations/cloud-aws.html

I guess it has been removed or replaced

### CODE REVIEW

**Comment by pdecat** [comment]
Resolves https://github.com/claranet/terraform-signalfx-integrations/issues/72

File: cloud/aws/variables.tf [hunk 1]
```diff
@@ -123,7 +123,6 @@ variable "included_services" {
     "AWS/EKS",
     "AWS/ElastiCache",
     "AWS/ElasticBeanstalk",
-    "AWS/ElasticInterface",
```
Comment (pdecat) [inline cloud/aws/variables.tf:126]: It was apparently a typo for `ElasticInference`, cf. https://github.com/splunk/public-o11y-docs/commit/12407dfccd43e523816e9bba4391b34d254b749f

Comment (pdecat) [reply cloud/aws/variables.tf:126]: It was a typo in Splunk's doc.

Comment (Tulux) [reply cloud/aws/variables.tf:126]: Got it, see 6b24b62

File: cloud/aws/variables.tf [hunk 2]
```diff
@@ -123,8 +123,8 @@ variable "included_services" {
...
     "AWS/ElasticMapReduce",
+    "AWS/ElasticInference",
```
Comment (pdecat) [inline cloud/aws/variables.tf:127]: Please keep this list sorted.

**Review by pdecat — APPROVED** [review]
LGTM!

**Pull request merged**